### PR TITLE
Bruno

### PR DESCRIPTION
**Pull request by brunoboto96**
Hey @ollymarchington @toderasc-corina   can you review this? Run bock2 and press ctrl-a to switch screens .
Also added requirements.txt ( run pip install -r Requirements.txt )
Run calendartest.py it works, you will need your own credentials.json maybe from: https://developers.google.com/calendar/quickstart/python

### CODE REVIEW

File: Bock2.py [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+    def change1(self, *args):
+        self.tk.withdraw()
+        self.tk2.deiconify()
+        return
+
+    def change2(self, *args):
+        self.tk2.withdraw()
+        self.tk.deiconify()
+        return
```
Comment (brunoboto96) [inline Bock2.py:256]: withdraw() hides
deiconify() shows

File: Requirements.txt [hunk 2]
```diff
@@ -1,2 +1,7 @@
+feedparser==5.2.1
+google_auth_oauthlib==0.4.1
 pyowm==2.10.0
-requests==2.21.0
\ No newline at end of file
+requests==2.21.0
+google_api_python_client==1.7.11
+Pillow==7.0.0
+protobuf==3.11.3
```
Comment (brunoboto96) [inline Requirements.txt]: mostly google api calendar

File: calendartest.py [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+                                          maxResults=10, singleEvents=True,
+                                          orderBy='startTime').execute()
+    events = events_result.get('items', [])
+
+    if not events:
+        print('No upcoming events found.')
+    for event in events:
+        start = event['start'].get('dateTime', event['start'].get('date'))
+        print(start, event['summary'])
+        self.Label2.configure(text=start+event['summary'])
```
Comment (brunoboto96) [inline calendartest.py:161]: google api demo code

Comment (brunoboto96) [inline calendartest.py:161]: sends event to gui

File: Bock2.py [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+
+        # fullscreen
+        self.toggle_fullscreen()  # start in fullscreen
+
+        self.change2()
+        # ctrl+f to toggle fullscreen
+        self.tk.bind("<Control-f>", self.toggle_fullscreen)
+        self.tk.bind("<Escape>", self.end_fullscreen)
+        self.tk.bind("<Control-a>", self.change1)
+        self.tk2.bind("<Control-a>", self.change2)
```
Comment (brunoboto96) [inline Bock2.py:215]: Then we just change this to left arrow or right arrow

File: Bock2.py (hunk 1 above)
Comment (brunoboto96) [reply Bock2.py:256]: I can try to mess with the position and transparency instead, but this does the job

File: credentials.json [hunk 5]
```diff
@@ -0,0 +1 @@
+{"installed":{"client_id":"283723347574-0vg3qol661e69n5fu95f1ulij5rjlghb.apps.googleusercontent.com","project_id":"quickstart-1583547139681","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"0ojUEYb0w6bTwZyaXVKWzWa0","redirect_uris":["urn:ietf:wg:oauth:2.0:oob","http://localhost"]}}
```
Comment (brunoboto96) [inline credentials.json:1]: this is my credential, feel free to use your own, I just added one window to test so it's just  the last slot

**Review by brunoboto96 — COMMENTED** [review]
line by line on my changes

**Review by ollymarchington — APPROVED** [review]

**Pull request merged**